From 16beefaf0b2892029fd067093fa498843d84ab0a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Lureau?= <marcandre.lureau@gmail.com>
Date: Wed, 26 Feb 2014 11:55:50 +0100
Subject: [PATCH] spice: do not open in fullscreen with
 CONTROLLER_AUTO_DISPLAY_RES

This flag is always set when using the rhevm user portal. Best is
probably to ignore it, now that fullscreen has simplified unique
behaviour.
---
 src/remote-viewer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/remote-viewer.c b/src/remote-viewer.c
index 0c6067c..2cb8925 100644
--- a/src/remote-viewer.c
+++ b/src/remote-viewer.c
@@ -561,7 +561,7 @@ spice_ctrl_notified(SpiceCtrlController *ctrl,
         virt_viewer_app_set_title(app, g_value_get_string(&value));
     } else if (g_str_equal(pspec->name, "display-flags")) {
         guint flags = g_value_get_uint(&value);
-        gboolean fullscreen = !!(flags & (CONTROLLER_SET_FULL_SCREEN | CONTROLLER_AUTO_DISPLAY_RES));
+        gboolean fullscreen = !!(flags & CONTROLLER_SET_FULL_SCREEN);
         g_object_set(G_OBJECT(self), "fullscreen", fullscreen, NULL);
     } else if (g_str_equal(pspec->name, "menu")) {
         spice_ctrl_menu_updated(self);
